src/main/java/mekanism/common/FuelHandler.java

```diff
@@ -56,7 +58,7 @@ public FuelGas(int duration, double energyDensity)
 		public FuelGas(IFuel bcFuel)
 		{
 			burnTicks = bcFuel.getTotalBurningTime() / Fluid.BUCKET_VOLUME;
-			energyPerTick = bcFuel.getPowerPerCycle() * general.FROM_RF;
+			energyPerTick = bcFuel.getPowerPerCycle() * MjAPI.MJ;
```

Comment:
I'm not familiar with mekanism's joule system, however getPowerPerCycle will return something like "6000000" (6 * MjApi.MJ). Currently the conversion from RF to MJ is `power = rf * MjApi.MJ / 20`, so I don't think you can convert exactly if 1J = 2.5 RF